ColorCoder.hpp

```diff
@@ -0,0 +1,39 @@
+namespace TelecommunicationsColorCoder
```

Comment:
This file is called `ColorCoder` but it doesn't code any colors - that functionality is in main.cpp